### DISCUSSION TITLE
The Re-render function not working on Angular 18

### DISCUSSION THREAD

**Discussion by 4factories** [l-lin/angular-datatables#1800] [Q&A]
Hi, I am using Angular 18 and I am trying to re-render the table after initialization, but it's not working. I need to reload because some rows may be deleted and I need to update the list with a backend call.

This is my code:
https://github.com/4factories/test/blob/9614a642838ba636236fefb247a29d79f1bbc987/lotto-stampo-list.component.html
https://github.com/4factories/test/blob/9614a642838ba636236fefb247a29d79f1bbc987/lotto-stampo-list.component.ts

Up to Angular 16 this method worked.

**Comment by 000geid** [comment]
any workaround for this?
